src/ocm/components/clusterConfiguration/operators/utils.tsx

```diff
@@ -0,0 +1,26 @@
+import {
+  OPERATOR_NAME_LVM,
+  Cluster,
+  FeatureSupportLevelData,
+  ExposedOperatorName,
+  OPERATOR_NAME_LVMS,
+} from '../../../../common';
+
+export const getActualLVMOperatorName = ({
+  openshiftVersion,
+  featureSupportLevel,
+  operator = OPERATOR_NAME_LVM,
+}: {
+  openshiftVersion: Cluster['openshiftVersion'];
+  featureSupportLevel: FeatureSupportLevelData;
+  operator?: ExposedOperatorName;
+}) => {
+  if (
```

Comment:
Since we are using this function in cases where `operator` is not LVM, the code has a case for which it doesn't work.

If `operator !== OPERATOR_NAME_LVM`, it needs to return that same operator name.